python/test/unit/language/print_helper.py

```diff
@@ -128,12 +136,18 @@ def test_print(func: str, data_type: str, device: str):
         kernel_device_print_hex[(1, )](x, y, num_warps=num_warps, BLOCK=N)
     elif func == "device_print_pointer":
         kernel_print_pointer[(1, )](x, y, num_warps=num_warps, BLOCK=N)
+    elif func == "device_print_2d_tensor":
```

Comment:
Is this new test added only passing after moving to COv5?

Reply:
It passes on COv4 but crashes on COv5 when using Rocm6.1 and below. So I thought it might be valuable. I can also split it into 2 separate PRs.